-f44e76:~# logread
Fri Sep  3 03:42:51 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:42:51 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:42:55 2021 user.info QM[1926]: <INFO>      MQTT: Connecting to: opensync-mqtt-broker-nola-02.cicd.lab.wlan.tip.build:1883:3c2c99f44e77
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2327}
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2328}
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:42:56 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:42:56 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:42:56 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:42:56 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:42:56 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>       MQTT: Connection failed: opensync-mqtt-broker-nola-02.cicd.lab.wlan.tip.build:1883:3c2c99f44e77: Error Lookup error.
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_enqueue_and_reply
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_write_res: b:48
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: incomplete 0/56 req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:0 r:0
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_new ctx:0 fd:11
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:818 r:818
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_handle_req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: complete from:SM c:2 to:null dt:2 ds:762 b:818
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_enqueue_and_reply
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_write_res: b:48
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: incomplete 0/56 req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:0 r:0
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Not connected, will reconnect in 0 secs
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_new ctx:0 fd:11
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:56 r:56
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_handle_req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: complete from:SM c:1 to:null dt:0 ds:0 b:56
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_enqueue_and_reply
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_write_res: b:48
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: incomplete 0/56 req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:0 r:0
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_new ctx:0 fd:11
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:7022 r:7022
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_handle_req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: complete from:SM c:2 to:null dt:2 ds:6966 b:7022
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_enqueue_and_reply
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_write_res: b:48
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_conn_parse_req: incomplete 0/56 req
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MISC: qm_async_callback ctx:0 fd:11 t:0 r:0
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Connecting to opensync-mqtt-broker-nola-02.cicd.lab.wlan.tip.build ...
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <INFO>      MQTT: Connecting to: opensync-mqtt-broker-nola-02.cicd.lab.wlan.tip.build:1883:3c2c99f44e77
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>  TRACEBACK: <TRACE>     MQTT: Reinit: cafile=/var/run/openvswitch/certs/ca.pem capath=(null) certfile=/var/run/openvswitch/certs/client.pem keyfile=/var/run/openvswitch/certs/client_dec.key pwcb=0 certreqs=1 tls=tlsv1.2 ciphers=DHE-DSS-AES128-SHA256:DHE-RSA-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384
Fri Sep  3 03:43:00 2021 user.err QM[1926]: <ERR>       MAIN: Connecting.
Fri Sep  3 03:43:00 2021 daemon.info lldpd[1521]: interface 255 does not have a name or an address, skip
Fri Sep  3 03:43:00 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>       MAIN: Processing 5G radio0 off-chan 149 survey report (Failed to scan)
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan1: added scan context
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MISC: wlan1: scan trigger returned 1
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan1: scan started
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan1: scan completed
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan1: calling context cb
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MAIN: Fetched 2.4G radio1 on-chan neighbor reporting context
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processing 2.4G radio1 on-chan neighbor report through survey
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MISC: wlan1: scan dump returned 1, list 1
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan scan records new 0 total 0
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 1 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 2 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 3 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 4 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 5 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 2338065 busy 47981 tx 46896 rx 344 rx_self 16 chan 6 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 7 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 8 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 9 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 10 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 11 type 2
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey convert scan_type 2 chan 6 duration_new:2338065 duration_old:2288024 busy_new 47981 busy_old:46990 tx_new 46896 tx_old:45910 rx_new 344 rx_old:344 rx_self_new 16 rx_self_old:16
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan 6 survey percent {busy=1 tx=1 self=0 rx=0 ext=0 noise=181 duration=50041}
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan survey records 4
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: sm_scan_schedule_cb. Scan done. Deleting scan_ctx. 0x14b5d10. 2.4G on-chan 6
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: sm_scan_schedule_cb. Schedule next scan request. 0xb6da24e0. 5G off-chan 149
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MAIN: Scheduled 5G off-chan 149 scan
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: delete scan context
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: added scan context
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MISC: wlan0: scan trigger returned 1
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: scan started
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: scan aborted
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: calling context cb
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>       MAIN: Flushed scheduled 5G off-chan 149 scan queues
Fri Sep  3 03:43:01 2021 user.err SM[6235]: <ERR>       MAIN: Flushed scheduled 2.4G off-chan 1 scan queues
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2329}
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2330}
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:01 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:01 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:01 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:01 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:01 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>       MISC: radio1: failed to set ht/hwmode
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "WM - update, if_name==wlan0"}, {"table": "Wifi_Master_State", "op": "update", "where": [["if_name", "==", "wlan0"]], "row": {"port_state": "active"}}], "id": 576996}
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table Wifi_Master_State update result: [{}, {"count": 0}] off: 0
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table Wifi_Master_State update count: 0
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: OVSDB: Update table 'Wifi_Master_State' where '[["if_name", "==", "wlan0"]]': no match
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: wlan0: updated (0) master state to 'active'
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: wlan0@radio0: changed, configuring
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Wifi_RRM_Config", "op": "select", "where": [["freq_band", "==", "5G"]]}], "id": 576997}
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": [{"_version": ["uuid", "41178f58-85e5-461e-8f5a-3a8ea8763a8d"], "freq_band": "5G", "probe_resp_threshold": -90, "min_load": 40, "mcast_rate": 0, "client_disconnect_threshold": -90, "noise_floor_thresh": -75, "_uuid": ["uuid", "9035ec2e-50f6-42ac-8fed-0558b895378a"], "noise_floor_time": 180, "backup_channel": 153, "beacon_rate": 0, "snr_percentage_drop": 30, "non_wifi_time": 180, "non_wifi_thresh": 50, "obss_hop_mode": 1}]}]
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where return: [{"_version": ["uuid", "41178f58-85e5-461e-8f5a-3a8ea8763a8d"], "freq_band": "5G", "probe_resp_threshold": -90, "min_load": 40, "mcast_rate": 0, "client_disconnect_threshold": -90, "noise_floor_thresh": -75, "_uuid": ["uuid", "9035ec2e-50f6-42ac-8fed-0558b895378a"], "noise_floor_time": 180, "backup_channel": 153, "beacon_rate": 0, "snr_percentage_drop": 30, "non_wifi_time": 180, "non_wifi_thresh": 50, "obss_hop_mode": 1}]
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MISC: wireless: created wlan0
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: wlan0: cancelling scheduled work wm2_vconf_recalc
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: radio1: running scheduled work wm2_rconf_recalc
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: radio1: recalculating
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Wifi_Radio_Config", "op": "select", "where": [["if_name", "==", "radio1"]]}], "id": 576998}
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": [{"dfs_demo": ["set", []], "if_name": "radio1", "temperature_control": ["map", []], "tx_power": 18, "rx_chainmask": 0, "thermal_downgrade_temp": ["set", []], "ht_mode": "HT20", "bcn_int": 100, "zero_wait_dfs": ["set", []], "thermal_tx_chainmask": ["set", []], "hw_mode": "auto", "enabled": true, "channel_sync": ["set", []], "thermal_shutdown": ["set", []], "thermal_upgrade_temp": ["set", []], "_uuid": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "custom_options": ["map", [["max_clients", "100"]]], "_version": ["uuid", "279a82e3-b6e2-4817-adf5-4a6beba888f5"], "freq_band": "2.4G", "hw_type": "ath10k", "thermal_integration": ["set", []], "channel_mode": ["set", []], "vif_configs": ["uuid", "8e851ec2-8a40-4d04-8ff6-7afe3e048767"], "country": "US", "tx_chainmask": 0, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6}]}]
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where return: [{"dfs_demo": ["set", []], "if_name": "radio1", "temperature_control": ["map", []], "tx_power": 18, "rx_chainmask": 0, "thermal_downgrade_temp": ["set", []], "ht_mode": "HT20", "bcn_int": 100, "zero_wait_dfs": ["set", []], "thermal_tx_chainmask": ["set", []], "hw_mode": "auto", "enabled": true, "channel_sync": ["set", []], "thermal_shutdown": ["set", []], "thermal_upgrade_temp": ["set", []], "_uuid": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "custom_options": ["map", [["max_clients", "100"]]], "_version": ["uuid", "279a82e3-b6e2-4817-adf5-4a6beba888f5"], "freq_band": "2.4G", "hw_type": "ath10k", "thermal_integration": ["set", []], "channel_mode": ["set", []], "vif_configs": ["uuid", "8e851ec2-8a40-4d04-8ff6-7afe3e048767"], "country": "US", "tx_chainmask": 0, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6}]
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Wifi_Radio_State", "op": "select", "where": [["if_name", "==", "radio1"]]}], "id": 576999}
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": [{"dfs_demo": ["set", []], "if_name": "radio1", "thermal_downgraded": ["set", []], "temperature_control": ["map", []], "mac": "3c:2c:99:f4:4e:78", "bcn_int": 100, "thermal_tx_chainmask": ["set", []], "allowed_channels": ["set", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]], "radio_config": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "thermal_shutdown": ["set", []], "channel_sync": ["set", []], "_uuid": ["uuid", "2706c307-9a77-46a8-b143-059fe0f3a4e4"], "hw_type": ["set", []], "vif_states": ["uuid", "238c268e-bc47-4ad7-8b28-2fb5d2588920"], "radar": ["map", []], "country": "US", "tx_chainmask": 15, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6, "tx_power": 18, "rx_chainmask": 15, "ht_mode": "HT20", "thermal_downgrade_temp": ["set", []], "hw_mode": "11n", "zero_wait_dfs": ["set", []], "enabled": true, "hw_params": ["map", []], "thermal_upgrade_temp": ["set", []], "channels": ["map", [["allowed", "1,2,
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where return: [{"dfs_demo": ["set", []], "if_name": "radio1", "thermal_downgraded": ["set", []], "temperature_control": ["map", []], "mac": "3c:2c:99:f4:4e:78", "bcn_int": 100, "thermal_tx_chainmask": ["set", []], "allowed_channels": ["set", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]], "radio_config": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "thermal_shutdown": ["set", []], "channel_sync": ["set", []], "_uuid": ["uuid", "2706c307-9a77-46a8-b143-059fe0f3a4e4"], "hw_type": ["set", []], "vif_states": ["uuid", "238c268e-bc47-4ad7-8b28-2fb5d2588920"], "radar": ["map", []], "country": "US", "tx_chainmask": 15, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6, "tx_power": 18, "rx_chainmask": 15, "ht_mode": "HT20", "thermal_downgrade_temp": ["set", []], "hw_mode": "11n", "zero_wait_dfs": ["set", []], "enabled": true, "hw_params": ["map", []], "thermal_upgrade_temp": ["set", []], "channels": ["map", [["allowed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'tx_chainmask' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'rx_chainmask' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'hw_mode' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: changed (forced=0)
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'tx_chainmask' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'rx_chainmask' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'hw_mode' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: changed (forced=0)
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'tx_chainmask' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'rx_chainmask' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'hw_mode' changed
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: changed (forced=0)
Fri Sep  3 03:43:02 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: radio1: changed, configuring
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2331}
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2332}
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:06 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:06 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:06 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:06 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:06 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:10 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip
Fri Sep  3 03:43:10 2021 daemon.info lldpd[1521]: interface 255 does not have a name or an address, skip
Fri Sep  3 03:43:10 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>       MAIN: Processing 5G radio0 off-chan 149 survey report (Failed to scan)
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan1: calling context cb
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MAIN: Fetched 2.4G radio1 on-chan neighbor reporting context
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processing 2.4G radio1 on-chan neighbor report through survey
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MISC: wlan1: scan dump returned 1, list 1
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan scan records new 0 total 0
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 1 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 2 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 3 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 4 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 5 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 2348286 busy 48185 tx 47098 rx 344 rx_self 16 chan 6 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 7 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 8 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 48 busy 0 tx 0 rx 0 rx_self 0 chan 9 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 10 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey entry dur 42 busy 0 tx 0 rx 0 rx_self 0 chan 11 type 2
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey convert scan_type 2 chan 6 duration_new:2347754 duration_old:2338065 busy_new 48174 busy_old:47981 tx_new 47088 tx_old:46896 rx_new 344 rx_old:344 rx_self_new 16 rx_self_old:16
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan 6 survey percent {busy=1 tx=1 self=0 rx=0 ext=0 noise=181 duration=9689}
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: Survey convert scan_type 2 chan 6 duration_new:2348286 duration_old:2347754 busy_new 48185 busy_old:48174 tx_new 47098 tx_old:47088 rx_new 344 rx_old:344 rx_self_new 16 rx_self_old:16
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan 6 survey percent {busy=2 tx=1 self=0 rx=0 ext=0 noise=181 duration=532}
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: Processed 2.4G radio1 on-chan survey records 7
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: sm_scan_schedule_cb. Scan done. Deleting scan_ctx. 0x14b5f00. 2.4G on-chan 6
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MAIN: sm_scan_schedule_cb. Schedule next scan request. 0x14b55c0. 5G off-chan 149
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MAIN: Scheduled 5G off-chan 149 scan
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: delete scan context
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: added scan context
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>     MISC: wlan0: scan trigger returned 1
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: scan started
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>    SCHED: Task 1 reschedule requested (same 7000 ms)
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <TRACE>    SCHED: Task 3 reschedule requested (same 7000 ms)
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: scan aborted
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>  TRACEBACK: <DEBUG>     MISC: wlan0: calling context cb
Fri Sep  3 03:43:11 2021 user.err SM[6235]: <ERR>       MAIN: Flushed scheduled 5G off-chan 149 scan queues
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2333}
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2334}
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:11 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:11 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:11 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:11 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:11 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:15 2021 user.info SM[6235]: <INFO>      MISC: RADIUS PROBE
Fri Sep  3 03:43:15 2021 user.info SM[6235]: <INFO>      MISC: RADIUS PROBE
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2335}
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2336}
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:16 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:16 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:16 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:16 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:16 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2337}
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2338}
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:21 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:21 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:21 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:21 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:21 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2339}
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2340}
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:26 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:26 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:26 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:26 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:26 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2341}
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2342}
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:31 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:31 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:31 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:31 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:31 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>       MISC: radio1: failed to set ht/hwmode
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "WM - update, if_name==wlan0"}, {"table": "Wifi_Master_State", "op": "update", "where": [["if_name", "==", "wlan0"]], "row": {"port_state": "active"}}], "id": 577078}
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table Wifi_Master_State update result: [{}, {"count": 0}] off: 0
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table Wifi_Master_State update count: 0
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: OVSDB: Update table 'Wifi_Master_State' where '[["if_name", "==", "wlan0"]]': no match
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: wlan0: updated (0) master state to 'active'
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: wlan0@radio0: changed, configuring
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Wifi_RRM_Config", "op": "select", "where": [["freq_band", "==", "5G"]]}], "id": 577079}
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": [{"_version": ["uuid", "41178f58-85e5-461e-8f5a-3a8ea8763a8d"], "freq_band": "5G", "probe_resp_threshold": -90, "min_load": 40, "mcast_rate": 0, "client_disconnect_threshold": -90, "noise_floor_thresh": -75, "_uuid": ["uuid", "9035ec2e-50f6-42ac-8fed-0558b895378a"], "noise_floor_time": 180, "backup_channel": 153, "beacon_rate": 0, "snr_percentage_drop": 30, "non_wifi_time": 180, "non_wifi_thresh": 50, "obss_hop_mode": 1}]}]
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where return: [{"_version": ["uuid", "41178f58-85e5-461e-8f5a-3a8ea8763a8d"], "freq_band": "5G", "probe_resp_threshold": -90, "min_load": 40, "mcast_rate": 0, "client_disconnect_threshold": -90, "noise_floor_thresh": -75, "_uuid": ["uuid", "9035ec2e-50f6-42ac-8fed-0558b895378a"], "noise_floor_time": 180, "backup_channel": 153, "beacon_rate": 0, "snr_percentage_drop": 30, "non_wifi_time": 180, "non_wifi_thresh": 50, "obss_hop_mode": 1}]
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MISC: wireless: created wlan0
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: wlan0: cancelling scheduled work wm2_vconf_recalc
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: radio1: running scheduled work wm2_rconf_recalc
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: radio1: recalculating
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Wifi_Radio_Config", "op": "select", "where": [["if_name", "==", "radio1"]]}], "id": 577080}
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": [{"dfs_demo": ["set", []], "if_name": "radio1", "temperature_control": ["map", []], "tx_power": 18, "rx_chainmask": 0, "thermal_downgrade_temp": ["set", []], "ht_mode": "HT20", "bcn_int": 100, "zero_wait_dfs": ["set", []], "thermal_tx_chainmask": ["set", []], "hw_mode": "auto", "enabled": true, "channel_sync": ["set", []], "thermal_shutdown": ["set", []], "thermal_upgrade_temp": ["set", []], "_uuid": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "custom_options": ["map", [["max_clients", "100"]]], "_version": ["uuid", "279a82e3-b6e2-4817-adf5-4a6beba888f5"], "freq_band": "2.4G", "hw_type": "ath10k", "thermal_integration": ["set", []], "channel_mode": ["set", []], "vif_configs": ["uuid", "8e851ec2-8a40-4d04-8ff6-7afe3e048767"], "country": "US", "tx_chainmask": 0, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6}]}]
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where return: [{"dfs_demo": ["set", []], "if_name": "radio1", "temperature_control": ["map", []], "tx_power": 18, "rx_chainmask": 0, "thermal_downgrade_temp": ["set", []], "ht_mode": "HT20", "bcn_int": 100, "zero_wait_dfs": ["set", []], "thermal_tx_chainmask": ["set", []], "hw_mode": "auto", "enabled": true, "channel_sync": ["set", []], "thermal_shutdown": ["set", []], "thermal_upgrade_temp": ["set", []], "_uuid": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "custom_options": ["map", [["max_clients", "100"]]], "_version": ["uuid", "279a82e3-b6e2-4817-adf5-4a6beba888f5"], "freq_band": "2.4G", "hw_type": "ath10k", "thermal_integration": ["set", []], "channel_mode": ["set", []], "vif_configs": ["uuid", "8e851ec2-8a40-4d04-8ff6-7afe3e048767"], "country": "US", "tx_chainmask": 0, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6}]
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Wifi_Radio_State", "op": "select", "where": [["if_name", "==", "radio1"]]}], "id": 577081}
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": [{"dfs_demo": ["set", []], "if_name": "radio1", "thermal_downgraded": ["set", []], "temperature_control": ["map", []], "mac": "3c:2c:99:f4:4e:78", "bcn_int": 100, "thermal_tx_chainmask": ["set", []], "allowed_channels": ["set", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]], "radio_config": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "thermal_shutdown": ["set", []], "channel_sync": ["set", []], "_uuid": ["uuid", "2706c307-9a77-46a8-b143-059fe0f3a4e4"], "hw_type": ["set", []], "vif_states": ["uuid", "238c268e-bc47-4ad7-8b28-2fb5d2588920"], "radar": ["map", []], "country": "US", "tx_chainmask": 15, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6, "tx_power": 18, "rx_chainmask": 15, "ht_mode": "HT20", "thermal_downgrade_temp": ["set", []], "hw_mode": "11n", "zero_wait_dfs": ["set", []], "enabled": true, "hw_params": ["map", []], "thermal_upgrade_temp": ["set", []], "channels": ["map", [["allowed", "1,2,
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where return: [{"dfs_demo": ["set", []], "if_name": "radio1", "thermal_downgraded": ["set", []], "temperature_control": ["map", []], "mac": "3c:2c:99:f4:4e:78", "bcn_int": 100, "thermal_tx_chainmask": ["set", []], "allowed_channels": ["set", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]], "radio_config": ["uuid", "35197691-67b4-4909-9460-4763862b44dc"], "thermal_shutdown": ["set", []], "channel_sync": ["set", []], "_uuid": ["uuid", "2706c307-9a77-46a8-b143-059fe0f3a4e4"], "hw_type": ["set", []], "vif_states": ["uuid", "238c268e-bc47-4ad7-8b28-2fb5d2588920"], "radar": ["map", []], "country": "US", "tx_chainmask": 15, "fallback_parents": ["map", []], "hw_config": ["map", []], "channel": 6, "tx_power": 18, "rx_chainmask": 15, "ht_mode": "HT20", "thermal_downgrade_temp": ["set", []], "hw_mode": "11n", "zero_wait_dfs": ["set", []], "enabled": true, "hw_params": ["map", []], "thermal_upgrade_temp": ["set", []], "channels": ["map", [["allowed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'tx_chainmask' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'rx_chainmask' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'hw_mode' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: changed (forced=0)
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'tx_chainmask' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'rx_chainmask' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'hw_mode' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: changed (forced=0)
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'tx_chainmask' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'rx_chainmask' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: 'hw_mode' changed
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <TRACE>     MAIN: radio1: changed (forced=0)
Fri Sep  3 03:43:32 2021 user.err WM[1934]: <ERR>  TRACEBACK: <DEBUG>     MAIN: radio1: changed, configuring
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2343}
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2344}
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:36 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:36 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:36 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:36 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:36 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2345}
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2346}
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:41 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:41 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:41 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:41 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:41 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2347}
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2348}
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:46 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:46 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:46 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:46 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:46 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:50 2021 daemon.info lldpd[1521]: interface 255 does not have a name or an address, skip
Fri Sep  3 03:43:50 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip
Fri Sep  3 03:43:50 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip
Fri Sep  3 03:43:51 2021 daemon.info lldpd[1521]: interface 255 does not have a name or an address, skip
Fri Sep  3 03:43:51 2021 daemon.info lldpd[1521]: interface 255 does not have a name or an address, skip
Fri Sep  3 03:43:51 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip
Fri Sep  3 03:43:51 2021 daemon.info lldpd[1521]: interface 255 does not have a name or an address, skip
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>       MISC: redirector DNS lookup failed (Try again)
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: OVS_INIT r: timer t: 0 o: -1
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"table": "Connection_Manager_Uplink", "op": "select", "where": [["if_type", "==", "gre"]]}], "id": 2349}
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: query result: [{"rows": []}]
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: ovsdb_sync_select_where: result no rows
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: cm2_ovsdb_remove_unused_gre_interfaces Available gre links count = 0
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>     MISC: Found is_inarray()::argc=1|i=0|argv[i]=manager_addr
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <TRACE>    OVSDB: filter keep key: manager_addr
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node where: '' update: {"manager_addr": ""}
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: SYNC: Writing sync operation: {"method": "transact", "params": ["Open_vSwitch", {"op": "comment", "comment": "CM - update"}, {"table": "AWLAN_Node", "op": "update", "where": [], "row": {"manager_addr": ""}}], "id": 2350}
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update result: [{}, {"count": 1}] off: 0
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    OVSDB: Table AWLAN_Node update count: 1
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>     MISC: Clear manager addr
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <DEBUG>    EVENT: === Update s: TRY_RESOLVE r: state-change t: 0 o: 180
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:51 2021 user.err CM[1922]: <ERR>  TRACEBACK: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:51 2021 user.warn CM[1922]: <WARNING>  EVENT: State TRY_RESOLVE failure -> OVS_INIT
Fri Sep  3 03:43:51 2021 user.notice CM[1922]: <NOTICE>   EVENT: State OVS_INIT success -> TRY_RESOLVE
Fri Sep  3 03:43:51 2021 user.info CM[1922]: <INFO>     EVENT: Trying to resolve redirector: opensync-redirector-nola-02.cicd.lab.wlan.tip.build
Fri Sep  3 03:43:51 2021 user.info CM[1922]: <INFO>      MISC: sync resolving redirector 'ssl:opensync-redirector-nola-02.cicd.lab.wlan.tip.build:6643'
Fri Sep  3 03:43:52 2021 daemon.info lldpd[1521]: interface 254 does not have a name or an address, skip

